spec/models/resource_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
```

Comment:
I don't remember modifying any ruby. I don't think this should be here. Can you remove this file from the PR?